modules/rhcos-enabling-multipath-day-2.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="rhcos-enabling-multipath-day-2_{context}"]
+= Enabling multipathing with kernel arguments on {op-system}
```

Comment:
There is no {op-system-first} on this entire page. Not sure if that matters, though.

Reply:
Yes, I think we should use {op-system-first} on first instance here in the paragraph text. Thanks!